deploy/docker-compose/README

```diff
@@ -0,0 +1,153 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between local development of MAPs in a laptop using the MONAI Deploy App SDK, and a production-ready deployment. 
```

Comment:
NIT: Don't see the need to mention "in a laptop". A user can choose to develop app and MAP on a DGX. It is meant to provide version of MONAI Deploy platform that has most, of not all, of the functionalities of the full version without the dependencies on Kubernetes or Argo, and as such, without the enterprise grade requirements on scalability, performance, and availability.

Reply:
@dbericat ?

Reply:
Laptop has no technical meaning in this context. Do you mean MacBook, ChromeBook, Windows PC with no GPU. None of these would work. Even with the [MONAI Deploy definition of target env](https://github.com/Project-MONAI/monai-deploy/blob/main/guidelines/MONAI-Operating-Environments.md),  Desk/Laptop env Laptop is not supported by MONAI Deploy App SDK, period.

Reply:
Agreed. I captured on my review comment as:

"As described on the MONAI Operating Environments guideline, the journey from development to production usually requires multiple steps across different environments, operated by different teams and with different requirements.

MONAI Deploy Lite is designed to facilitate the testing and validation of MAPs in the early stages of this pipeline (i.e. workstation environment), where ease of use and time to get started is most important."

Reply:
Updated with @dbericat's feedback